test/integration/server-test.ts

```diff
@@ -97,6 +97,11 @@ test("POST / with push event payload (request.body already parsed)", (t) => {
   const server = http.createServer((req, res) => {
     req.once("data", (chunk) => dataChunks.push(chunk));
     req.once("end", () => {
+      /*
+       * @ts-ignore: Until getPayload @ts-ignore is fixed
+       * (https://github.com/dominguezcelada/webhooks.js/blob/b2162f4489efb9ee828cfd57e82bf98d12503c4b/src/middleware/get-payload.ts#L7-L8)
+       */
+      //@ts-ignore
```

Comment:
it's not really something that needs fixing, it's a use case with have to account for. `req.body` is set in some serverless environments, the parsing is happening outside of our control. 

We could amend the `@ts-ignore in both places
```suggestion
      // @ts-ignore account for serverless environments that provide a pre-parsed `req.body` object
```